packages/frontend/core/src/layouts/workspace-layout.tsx

```diff
@@ -133,7 +133,7 @@ function useLoadWorkspacePages() {
           .map(id => currentWorkspace.blockSuiteWorkspace.getPage(id))
           .filter((p): p is Page => !!p);
         pages.forEach(page => {
-          loadPage(page);
+          loadPage(page, 10);
```

Comment:
😊 Hey, i think here should be a small number, like `-10`, because according to the p-queue documentation `Operations with greater priority will be scheduled first.`

Here are lazy load pages so they should have a smaller priority

Reply:
ops, I thought smaller priority number means higher priority